awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -192,10 +193,16 @@ def create_reference_data(source_dir, env, content):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('this_kind', CLOUD_PROVIDERS)
-def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, mock_me):
+@pytest('this_kind', discover_available_cloud_provider_plugin_names)
```

Comment:
`@pytest` is a module, not a decorator, it's not really possible to call it or decorate with it. Also, I see no reason why passing a callable into a non-existing decorator call would work. This just needs to be deleted.
```suggestion
```